features/number.yml

```diff
@@ -0,0 +1,30 @@
+name: Number
+description: The Number object is used to store number and manipulate floating-point numbers, like 42, or -4.201
+spec: https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-number-objects
```

Comment:
So here is where I would add `group: primitive-types`, though that will require a new group file. It should probably have a comment that it's sort of a transitional thing, until we have feature composition sorted out.

The parent of the group is almost certainly `javascript`.